face/UnifiedFaceCollection/python/unified_face_collection.py

```diff
@@ -0,0 +1,346 @@
+import requests
+import sys
+import os
+import time
+import json
+
+project_root = os.path.abspath(os.path.join(os.getcwd(), "../../"))
+if project_root not in sys.path:
+    sys.path.append(project_root)
+from PersonDirectoryOperations.python.shared_functions import detect_faces, enlarge_bounding_box, get_image_dimensions
```

Comment:
Personally I would prefer having a duplicated implementation inside this sample. The cost to maintain these tiny helpers are pretty cheap, in comparison to the abrupt connection here. But it's up to you. 😊